certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -65,6 +65,8 @@ func (p *DeployableByOlmCheck) Validate(bundleRef certification.ImageReference)
 		return false, err
 	}
 
+	log.Debug("The operator Metadata is: ", *operatorData)
```

Comment:
Since I haven't seen a log statement of logging to a pointer, do we need a `%something in the quotes` or does go just log this properly and in a human readable format?

Reply:
Good catch. I'll update this to print a struct field/value.